tests/2d_fs_test.prm.fastscape.prm

```diff
@@ -0,0 +1,143 @@
+# This is a test on the input sea level function for the Fastscape Plugin. 
+# Sea level can be defined either as a constant value or as a user-defined function. 
+
+set Dimension                              = 2
```

Comment:
@xlia62 - Can you provide a high-level summary of the test in terms of what features it is testing, overall, design, etc?